uiplib/scrape/onlineFetch.py

```diff
@@ -1,20 +1,17 @@
 """Module to fetch wallpapers."""
 
-from threading import Thread
-
 from uiplib.scrape.scrape import get_images
 
 
-class onlineFetch(Thread):
-    """Generic thread module to download images."""
+class onlineFetch():
+    """Generic module to download images."""
 
     def __init__(self, url, directory, count):
-        """Initialize the fetch thread."""
-        Thread.__init__(self)
+        """Initialize the fetch."""
         self.url = url
         self.directory = directory
         self.count = count
 
     def run(self):
-        """Begin the download. Automatically called when a thread starts."""
+        """Begin the download. Automatically called when a Fetch starts."""
```

Comment:
or merge it in the init itself? no point of making a new function then is there? you'd also want to remove the start() call in other files right?